main.c

```diff
@@ -71,10 +74,60 @@ static struct kanshi_profile *match(struct kanshi_state *state,
 }
 
 
+static void exec_command(char *cmd) {
+	pid_t pid, child;
+	if ((pid = fork()) == 0) {
+		setsid();
+		sigset_t set;
+		sigemptyset(&set);
+		sigprocmask(SIG_SETMASK, &set, NULL);
+		if ((child = fork()) == 0) {
+			execl("/bin/sh", "/bin/sh", "-c", cmd, (void *)NULL);
+			fprintf(stderr, "Executing cmd '%s' failed: ", cmd);
+			perror("");
```

Comment:
It would be better to just `fprintf(stderr, "Executing cmd '%s' failed: %s", cmd, strerror(errno));`